lib/pf/api/jsonrpcclient.pm

```diff
@@ -177,6 +197,8 @@ sub curl {
     $curl->setopt(CURLOPT_NOSIGNAL, 1);
     $curl->setopt(CURLOPT_URL, $url);
     $curl->setopt(CURLOPT_HTTPHEADER, ['Content-Type: application/json-rpc',"Request: $function"]);
+    $curl->setopt(CURLOPT_CONNECTTIMEOUT_MS, $self->connect_timeout_ms // 0);
```

Comment:
It already defaults to 0.

Either 0 should be a constant used here and in the default value above or this should be removed so its not duplicate

Reply:
That is there to if connect_timeout_ms was set to undef.